File: data/imported/knowledge/cells/java/merged/snippets/snippet_077.java
@Test
    void CV_31_getCellsGetRowColAcceptsZeroBased() {
        Workbook wb = new Workbook();
        Cells cells = wb.getWorksheets().get(0).getCells();
        cells.get(0, 0).putValue(1);
        assertEquals(1, cells.get("A1").getValue());
        assertEquals(cells.get(0, 0).getValue(), cells.get("A1").getValue());
    }

File: data/imported/knowledge/cells/cpp/merged/snippets/snippet_077.cpp
TEST(WorkbookPortingTests, StyleFormattingPersistsThroughXlsxRoundTrip)
{
    TempDir temp;
    const auto path = temp.Path("style.xlsx");

    Workbook workbook;
    auto cell = workbook.GetWorksheets()[0].GetCells()["C4"];
    cell.PutValue(0.125);
    auto style = cell.GetStyle();
    ApplyVisibleStyle(style);
    cell.SetStyle(style);
    workbook.Save(path.string());

    Workbook loaded(path.string());
    auto loadedCell = loaded.GetWorksheets()[0].GetCells()["C4"];
    auto loadedStyle = loadedCell.GetStyle();

    EXPECT_EQ("0.00%", loadedStyle.GetNumberFormat());
    EXPECT_TRUE(loadedStyle.GetFont().GetBold());
    EXPECT_TRUE(loadedStyle.GetFont().GetItalic());
    EXPECT_EQ("Arial", loadedStyle.GetFont().GetName());
    EXPECT_EQ(FillPattern::Solid, loadedStyle.GetPattern());
    EXPECT_EQ("12.50%", loadedCell.GetDisplayStringValue());
}